app/views/placements/support/schools/mentors/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= content_for :page_title, sanitize("#{@mentor.full_name} - #{@school.name}") %>
+
+<%= render "placements/support/primary_navigation", current_navigation: :organisations %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: placements_support_school_mentors_path) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= @school.name %></span>
```

Comment:
For the support user, this is slightly different than for the normal user. It should be 
Mentors - school_name
<img width="729" alt="image" src="https://github.com/DFE-Digital/itt-mentor-services/assets/44073106/ba515af9-599b-4bee-9ee9-3fd82240e48c">

Reply:
resolved in [f66b7fb](https://github.com/DFE-Digital/itt-mentor-services/pull/274/commits/f66b7fbdbaca1bed6ce46733c6c8777cdf3e27aa)